lib/OpenQA/Utils.pm

```diff
@@ -44,6 +45,7 @@ $VERSION = sprintf "%d.%03d", q$Revision: 1.12 $ =~ /(\d+)/g;
   &human_readable_size
   &locate_asset
   &job_groups_and_parents
+  wait_with_progress
```

Comment:
Btw, i found this (and the links within have also useful information) https://www.learning-perl.com/2013/05/why-we-teach-the-subroutine-ampersand/